docs/en/operation/StressBench.md

```diff
@@ -24,90 +24,91 @@ the Alluxio system. The following benchmark tools are currently supported:
 * `StressClientBench` - A benchmark tool measuring the IO performance of Alluxio through the client.
 * `StressMasterBench` - A benchmark tool measuring the master performance of Alluxio.
 * `StressWorkerBench` - A benchmark tool measuring the IO performance of Alluxio with single node.
-* `UfsIOBench` - A benchmark tool measuring the IO throughput between the Alluxio cluster and the
-  UFS.
-
-## High-Level Design
-StressBench is a job that runs with the job service. Therefore, the design follows what the job
-service supports. There are a few components for StressBench as follows:
+* `UfsIOBench` - A benchmark tool measuring the IO throughput between the Alluxio cluster and the UFS.
 
-### Client CLI
+## Examples
+### Basic Examples
 The user-facing CLI program allows you to invoke the StressBench tasks through the command line.
+Run a benchmark by its class name using the `runClass` command (there is a feature request to implement a generic CLI for StressBench [14083](https://github.com/Alluxio/alluxio/issues/14083)).
 
-**Invoking the benchmarks**
-
-You can run a benchmark by its class name using the `runClass` command. For example,
+For example,
 
 ```console
 $ bin/alluxio runClass alluxio.stress.cli.StressMasterBench --help
 ```
 
-invokes the `StressMasterBench` and prints its usage.
+This command invokes the `StressMasterBench` and prints its usage.
+
+To test RPC throughput of Alluxio master (e.g., for certain RPCs):
+
+```console
+$ bin/alluxio runClass alluxio.stress.cli.GetPinnedFileIdsBench
+$ bin/alluxio runClass alluxio.stress.cli.RegisterWorkerBench
+$ bin/alluxio runClass alluxio.stress.cli.WorkerHeartbeatBench
+```
+
+To benchmark Alluxio FUSE performance
+```console
+$ bin/alluxio runClass alluxio.stress.cli.fuse.FuseIOBench
+```
+
+To benchamrk job service
+```console
+$ bin/alluxio runClass alluxio.stress.cli.StressJobServiceBench
+```
 
-**Common options**
+### Options
 
-Common options are options that all benchmarks accept:
+Here is the list of common options that all benchmarks accept:
 
+* `--bench-timeout`: the maximum time a benchmark should run for
 * `--cluster`: run the benchmark with job service.
 * `--cluster-limit`: specify how many job workers are used to run the benchmark in parallel. 
   Omit to use all available job workers.
-* `--cluster-start-delay`: specify the start delay before starting the benchmark, used to
-  synchronize the jobs.
-* `--java-opt`: additional JVM options
-* `--profile-agent`: profile agent to use during benchmark
-* `--bench-timeout`: the maximum time a benchmark should run for
+* `--cluster-start-delay`: specify the start delay before starting the benchmark, used to synchronize jobs.
+* `--java-opt`: specify additional JVM options
 * `--help`: print usage, description and benchmark-specific options of a benchmark
+* `--profile-agent`: profile agent to use during benchmark
```

Comment:
can you remove this profile agent option?

Reply:
```suggestion
```